src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  line-height: 0;
+}
+
+.header {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.way__home {
+  flex-grow: 1;
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+  padding: 10px;
+  padding-left: 50px;
+}
+
+.nav {
+  margin-right: 40px;
+}
+
+.nav__height {
+  height: 100%;
+  display: flex;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none ;
+  padding: 0 10px;
+  text-transform: uppercase;
+  height: 100%;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
move repeatable colors in the variables

Reply:
The repeatable colors are used for a specific class and :hover for another class, should i still remove it?

Reply:
yes, if you use it more then one time

Reply:
Done